### PR TITLE
[Console] CS fix

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | --
| License       | MIT

### CODE REVIEW

**Review by keradus — APPROVED** [review]

File: src/Symfony/Component/Console/Completion/CompletionInput.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public static function fromString(string $inputStr, int $currentIndex): self
      * Create an input based on an COMP_WORDS token list.
      *
      * @param string[] $tokens       the set of split tokens (e.g. COMP_WORDS or argv)
-     * @param          $currentIndex the index of the cursor (e.g. COMP_CWORD)
+     * @param int      $currentIndex the index of the cursor (e.g. COMP_CWORD)
```
Comment (wouterj) [inline src/Symfony/Component/Console/Completion/CompletionInput.php]: This is intentional. The type is optional and already given in de PHP code. We've done this in multiple locations (and I would even be in favor of removing all duplicate types from the PHPdoc)

**Review by wouterj — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Console/Completion/CompletionInput.php (hunk 1 above)
Comment (OskarStark) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: PHPStorm was complaining about it:
CleanShot 2024-01-03 at 14 43 40@2x

Comment (OskarStark) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: I think we should be content, in 5.4 only two more needs to be fixed, which I've done in this PR.
I also created a PR for 6.3
* #53391 

If both are accepted and (up)merged, I will check 6.4, 7.0 and 7.1

Comment (wouterj) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: To me, that's a bug in PHPstorm.

Comment (chalasr) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: @wouterj Do you have a link to a discussion about this? 
I consider such doc param to be incomplete and less readable: when reading a param's description, not having the type right before it requires to find and look at the corresponding parameter inside the signature to get the full picture

File: src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php [hunk 2]
```diff
@@ -92,9 +92,6 @@ public function prune()
         return $pruned;
     }
 
-    /**
-     * {@inheritdoc}
-     */
```
Comment (OskarStark) [inline src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php]: Those "inheritdoc" got removed by applying the fabbot patch, shall I revert?

File: src/Symfony/Component/Console/Completion/CompletionInput.php [hunk 3]
```diff
@@ -34,7 +34,7 @@ final class CompletionInput extends ArgvInput
     private $tokens;
     private $currentIndex;
     private $completionType;
-    private $completionName = null;
+    private $completionName;
```
Comment (OskarStark) [inline src/Symfony/Component/Console/Completion/CompletionInput.php]:
```suggestion
    private $completionName = null;
```

File: src/Symfony/Component/Console/Helper/ProgressIndicator.php [hunk 4]
```diff
@@ -129,8 +129,6 @@ public function advance()
 
     /**
      * Finish the indicator with message.
-     *
-     * @param $message
```
Comment (OskarStark) [inline src/Symfony/Component/Console/Helper/ProgressIndicator.php:133]: Useless

File: src/Symfony/Component/Console/Completion/CompletionInput.php (hunk 1 above)
Comment (wouterj) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: Sure, it has always been part of our coding standard:

> - Add PHPDoc blocks for classes, methods, and functions only when they add relevant information that does not duplicate the name, native type declaration or context (e.g. instanceof checks);

Discussed in the PR making the change: https://github.com/symfony/symfony/pull/41912#pullrequestreview-696453567

See also https://github.com/symfony/symfony/issues/46140 https://github.com/symfony/symfony/pull/46218 https://github.com/symfony/symfony/pull/46217 https://github.com/symfony/symfony/pull/44935 https://github.com/symfony/symfony/pull/45324 (a couple of these are rejected by you actually Robin 😉)

File: src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php (hunk 2 above)
Comment (keradus) [reply src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php]: any value to keep it? i would suggest to remove all.
and if anything, I would suggest to switch to `#[Override]` (probably as alignment for whole project, not this single place only)

File: src/Symfony/Component/Console/Completion/CompletionInput.php (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Component/Console/Completion/CompletionInput.php]: Fair enough :)

File: src/Symfony/Component/Console/Helper/ProgressIndicator.php (hunk 4 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Console/Helper/ProgressIndicator.php:133]: I think that's the only change we might wanna keep. See @wouterj's comment.
Also 5.4 is for bug fixes only, and this is not one.

Comment (OskarStark) [reply src/Symfony/Component/Console/Helper/ProgressIndicator.php:133]: Done

**Comment by nicolas-grekas** [comment]
Thank you @OskarStark.

**Pull request merged**